docker-gen.sh

```diff
@@ -62,9 +62,9 @@ done
 
 if [ -z "$IMAGE_NAME" ]; then
    if [ $DEV_MODE = 1 ]; then
-       IMAGE_NAME="omisegoimages/ewallet-builder:stable"
+       IMAGE_NAME="omisegoimages/ewallet-builder:v1.1"
    else
-       IMAGE_NAME="omisego/ewallet:latest"
+       IMAGE_NAME="omisego/ewallet:v1.1-dev"
```

Comment:
I bit torn about using `v1.1-dev` here because it means we have to change again when we release. Not sure if there’s a better way though :\

Reply:
Yeah same here. But considered that Mixfile also needs to be updated all I think it's still bare-able...